docs/css/bootstrap.min.css

```diff
@@ -1900,7 +1900,7 @@ samp {
 code {
     padding: 2px 4px;
```

Comment:
make this also with core-lib color 
<img width="342" alt="image" src="https://github.com/shay-te/core-lib/assets/5166849/d06f1e0c-f834-4fc0-8268-e562ca7a8fd1">